src/server/main.cc

```diff
@@ -11,14 +11,40 @@
 #include "src/proto/protocached.pb.h"
 #include "src/proto/protocached.grpc.pb.h"
 
-void RunServer() {
+// TODO(joejoevictor): Move this to separate header and implementation file
+class ProtocachedServiceImpl final : public protocached::Protocached::Service {
+  grpc::Status Get(
+    grpc::ServerContext* context,
+    const ::protocached::GetRequest* request,
+    protocached::GetResponse* response
+  ) override {
```

Comment:
what is this "override" syntax lol... i don't know anything abt cpp now...

Reply:
Just overriding the funtion implementation generated by grpc/protobuf. You could search for what it's overriding by looking for

::grpc::Status Protocached::Service::Get

in the output file bazel-out/k8-fastbuild/genfiles/src/proto/protocached.grpc.pb.cc in your workspace after you build the server.